holesky/README.md

```diff
@@ -56,28 +68,25 @@ In order to opt-in into EigenDA as an Operator, you must meet the following dele
 - The operator to churn out has less than 10.01% of the total stake
 
 Execute the following command to opt-in to EigenDA AVS:
-This command also downloads the latest SRS points if they don't exist on the node. The file is approximately 8GB in size and the opt-in process can some time to complete depending on the network bandwidth.
-
-Note: EigenDA enables the following Quorum settings.
-Quorums Maps:
```

Comment:
I think these are important information for operators, do we find it's not the case (e.g. operators generally know what is quorum 0 and 1)?

Reply:
That information was not deleted, it was simply moved.

"This command also downloads the latest SRS points" was moved

"Quorums Maps:" content was rephrased and moved b/c we haven't referred to the allowed quorum settings as "Maps" publicly. Although we could if you feel strongly the term "Maps" should be included?